src/publishers/snapcraft.js

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs-extra';
+import path from 'path';
+import Snapcraft from 'electron-installer-snap/snapcraft';
+
+import asyncOra from '../util/ora-handler';
+
+/**
+ * `forgeConfig.snapStore`:
+ * * `release`: comma-separated list of channels to release to
+ */
+export default async ({ dir, artifacts, forgeConfig }) => {
+  const snapArtifacts = artifacts.filter(artifact => artifact.endsWith('.snap'));
+
+  if (snapArtifacts.length === 0) {
+    throw 'No snap files to upload!';
```

Comment:
Let's try help users here.

`No snap files to upload, please ensure you added the "snap" target to your forge config`

Reply:
Both error messages have been updated.